frontend/test/metabase/scenarios/alert/email_alert.cy.spec.js

```diff
@@ -32,7 +32,7 @@ describe("scenarios > alert > email_alert", () => {
 
       cy.visit("/admin/settings/email");
       cy.findByText("SMTP Host");
-      setupLocalHostEmail();
+      setupEmail();
```

Comment:
I guess we're already using email in CI. Where do we start the SMTP server?

Reply:
We aren't... It looks like even though there's no SMTP server set up, Metabase is allowing people to make alerts- Does Metabse just check that information in the email setup space or does it actually check that the email is working before allowing users to create alerts? That's where I think the disconnect is happening.

Reply:
> Does Metabase just check that information in the email setup space or does it actually check that the email is working before allowing users to create alerts?

I guess we don't check either! I had thought that we validate the email settings before saving like we do for some auth options, but I just checked and we don't. I don't know whether SMTP has a good way to check credentials before sending an email, but the current state doesn't feel to broken IMO.

Given our lack of validation, I think we could update the default snapshot to have email configured. That could use an API call rather than button-clicking, but we'd still want to click buttons in the smoketest that also uses `setupEmail`. It's probably not worth doing all of that here, but I would swap this out for an API call since this is in a `beforeEach` and clicking is slower than sending a request.

Reply:
I just pushed up my attempt at making it an API call in this specific case, but it's not currently working. The code won't run when I use `-` in the `settings` map (ie. `email-smtp-host`), but turning them into a string gets me an error claiming `API endpoint does not exist`. Any suggestions?